pyproject.toml

```diff
@@ -1,14 +1,11 @@
-[build-system]
-requires = ["setuptools", "wheel"]
-build-backend = "setuptools.build_meta"
-
-
 [project]
 name = "sonata-cell-position"
-version = "2024.10.1"
+version = "0.0.0"
```

Comment:
how does versioning work now?

Reply:
This version field wasn't actually used. The versioning still works as before, using git tags that trigger the CI, and exporting the version as an env variable in the resulting Docker image.

Reply:
ah, thanks.